modules/serverless-ossm-v2x-jwt.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * serverless/networking/serverless-ossm-jwt.adoc
+
+[id="serverless-ossm-v2x-jwt_{context}"]
+= Using JSON Web Token authentication with {ProductShortName} 2.x and {ServerlessProductName}
+
+.Procedure
+
+. Create the `RequestAuthentication` resource in a serverless application namespace, that is a member in the `ServiceMeshMemberRoll` object,  by copying the following code into a YAML file:
++
+[source,yaml]
+----
+apiVersion: security.istio.io/v1beta1
+kind: RequestAuthentication
+metadata:
+  name: jwt-example
+spec:
+  jwtRules:
+  - issuer: testing@secure.istio.io
+    jwksUri: https://raw.githubusercontent.com/istio/istio/release-1.8/security/tools/jwt/samples/jwks.json
+----
+. Apply the `RequestAuthentication` resource:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+. Allow access to the `RequestAuthenticaton` resource from system pods for a serverless application namespace, which is a member in the `ServiceMeshMemberRoll` object, by copying the following `AuthorizationPolicy` resources into a YAML file:
++
+[source,yaml]
+----
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: allowlist-by-paths
+spec:
+  action: ALLOW
+  rules:
+  - to:
+    - operation:
+        paths:
+        - /metrics <1>
+        - /healthz <2>
+----
+<1> The path to collect metrics by system pod.
+<2> The path to probe by system pod.
+. Apply the `AuthorizationPolicy` resource YAML file:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+. Copy the following `AuthorizationPolicy` resources for a serverless application namespace, which is a member in the `ServiceMeshMemberRoll` object, into a YAML file:
```

Comment:
Here also, I think we should say that this should be done for **each** application namespace they're willing to integrate.